Test/Unit/Section/CustomerData/BoltCartTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2020 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Test\Unit\Section\CustomerData;
+
+use Bolt\Boltpay\Helper\Cart as CartHelper;
+use Bolt\Boltpay\Section\CustomerData\BoltCart;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class BoltCartTest
+ * @package Bolt\Boltpay\Test\Unit\Section\CustomerData
+ * @coversDefaultClass \Bolt\Boltpay\Section\CustomerData\BoltCart
+ */
+class BoltCartTest extends TestCase
+{
+    /**
+     * @var CartHelper
+     */
+    private $cartHelper;
+
+    /**
+     * @var BoltCart
+     */
+    private $boltCart;
+
+    /**
+     * @inheritdoc
+     */
+    public function setUp()
+    {
+        $this->cartHelper = $this->createPartialMock(CartHelper::class, ['calculateCartAndHints']);
+        $this->boltCart = (new ObjectManager($this))->getObject(
+            BoltCart::class,
+            [
+                'cartHelper' => $this->cartHelper
+            ]
+        );
+    }
+
+    /**
+     * @test
+     * @covers ::getSectionData
+     */
+    public function getSectionData()
+    {
+        $this->cartHelper->expects(self::once())->method('calculateCartAndHints')->willReturnSelf();
```

Comment:
Could you explain this line? Why `willReturnSelf`

Reply:
@johnmccombs1 Can you reply?

Reply:
@vitaliyreznikov 
Could you explain this line? 
=> This line is to test that the calculateCartAndHints method is called once whenever the getSectionData method is executed. 
Why willReturnSelf ?
=> Thanks for pointing out. I've removed "willReturnSelf".